integrationtests/utils.py

```diff
@@ -41,7 +41,7 @@
 MNIST_CONFIG_FILE = MODYNCLIENT_CONFIG_PATH / "mnist.yaml"
 DUMMY_CONFIG_FILE = MODYNCLIENT_CONFIG_PATH / "dummy.yaml"
 CLIENT_ENTRYPOINT = SCRIPT_PATH.parent.parent / "modynclient" / "client" / "modyn-client"
-NEW_DATASET_TIMEOUT = 600
+NEW_DATASET_TIMEOUT = 30
```

Comment:
Any reason for this change? I guess 600 is indeed a bit high...

Reply:
During debugging, I was annoyed that the tests always take so long even without any particular reason to wait 600s. Even 10s was fine for me.